pkg/test/framework/runtime/components/galley/native.go

```diff
@@ -194,7 +206,7 @@ func (c *nativeComponent) Close() (err error) {
 		c.client = nil
 	}
 	if c.server != nil {
-		err := multierror.Append(c.server.Close()).ErrorOrNil()
+		err := multierror.Append(c.server.ForceClose()).ErrorOrNil()
```

Comment:
Can you add a comment explaining why ForceClose is needed.

Reply:
added